packages/client/test/miner/pendingBlock.spec.ts

```diff
@@ -140,13 +146,16 @@ tape('[PendingBlock]', async (t) => {
     const parentBlock = await vm.blockchain.getLatestBlock()
     const payloadId = await pendingBlock.start(vm, parentBlock)
     t.equal(pendingBlock.pendingPayloads.length, 1, 'should set the pending payload')
-    const block = await pendingBlock.build(payloadId)
-    t.ok(block?.header.number.eqn(1), 'should have built block number 1')
+    const built = await pendingBlock.build(payloadId)
+    if (!built) return t.fail('pendingBlock did not return')
+    const [block, receipts] = built
+    t.ok(block.header.number.eqn(1), 'should have built block number 1')
```

Comment:
Can we change this to `t.equals(block.header.number.toNumber(), 1, 'should have built block number 1')` so it'll be easier to map over to develop?

Reply:
This line is existing so it's already mapped and changed on develop, if I change it here then there will be a merge error for the line on develop.